src/mongoLocker.ts

```diff
@@ -0,0 +1,14 @@
+import getMongoLocker, { LockKey, LockCallback, LockerOptions } from './getMongoLocker';
+import getCollection from "./getCollection";
+
+let lockerPromise: ReturnType<typeof getMongoLocker>;
+
+export default async function <T>(key: LockKey, callback: LockCallback<T>, options: LockerOptions = {}): Promise<T> {
```

Comment:
If possible I would avoid "export default" for new functionality as it often leads to situations where you need to add i.e. some constants or extend it with other methods.

Reply:
I agree with this in case of libraries or services, but don't see the need with single-purpose files / functions
All lodash is written this way :)

Reply:
ok, for this case it should be fine